serving/using-external-dns.md

```diff
@@ -0,0 +1,119 @@
+# Using ExternalDNS to automate DNS setup
```

Comment:
Just to make sure I understand, why is it necessary to automate DNS setup? It seems like publishing DNS records is a one-time thing you have to do when you first setup a custom domain.

Also, would it also be helpful to set up a static IP address for the Knative cluster?

Reply:
This is a good question. The benefits we can get from ExternalDNS are:
1. If user cannot (or does not want to) set up a static IP for Knative in their working environment, then user does not need to manually update their DNS record every time when they reinstall/restart Knative.
2. It provides a possibility for user to dynamically and programmatically control the Knative DNS.

ExternalDNS is not a strong requirement for Knative. But I think it is good to have an introduction for it.